src/DomDocuments/CustomersDocument.php

```diff
@@ -103,6 +104,39 @@ public function addCustomer(Customer $customer): void
                 // Add the full element
                 $financialElement->appendChild($element);
             }
+
+            //check if collectmandate should be set
+            if ($customer->getCollectMandate() !== null) {
+
```

Comment:
I know it's not all best practice here in this class, but could you try to see if you can use some of the convenience methods in `\PhpTwinfield\DomDocuments\BaseDocument`?